src/expr/src/refresh_schedule.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use mz_repr::Timestamp;
+use serde::{Deserialize, Serialize};
+use std::time::Duration;
```

Comment:
We usually separate the `std` imports into their own block.

Reply:
Yes, sorry, fixed!

Reply:
Super minor, but I think this is still outstanding!

Reply:
Sorry, I resolved it locally, just haven't pushed it yet, because I was waiting for a Nightly.